Week2/README.md

```diff
@@ -175,85 +188,95 @@ To learn more about this topic, check out the following:
 
 #### Inner Joins
 
-Let’s say we wanted to get a list of those students and the details of their teacher. This would be a perfect fit for an inner join, since an inner join returns records at the intersection of the two tables.
+Let’s say we wanted to get a list of students and the details of their teacher, dismissing students without a teacher.
+This would be a perfect fit for an inner join, since an inner join returns records at the intersection of the two tables.
 
 ```sql
-SELECT s.first_name, s.last_name, s.gender, s.grade, t.full_name
-FROM students s
-INNER JOIN teachers t
-ON s.teacher_number = t.teacher_number
+SELECT students.first_name, students.last_name, students.gender, students.grade, teachers.full_name
+FROM students
+INNER JOIN teachers
+ON students.teacher_number = teachers.teacher_number
 ```
 
-#### Self Joins
+The keyword `INNER` is optional and writing `JOIN teachers` will result in an inner join.
 
-A self join is a join in which a table is joined with itself (which is also called Unary relationships), especially when the table has a FOREIGN KEY which references its own PRIMARY KEY. To join a table itself means that each row of the table is combined with itself and with every other row of the table. In department, we want to get the employee's information with their direct manager. Here, each row in employees has a foreign key to itself as manager_id:
+#### Aliases
+
+When you join two tables there can be columns in both tables with the same name.
+To be explicit about the column you need you have to prefix the column with the table name like `table.column`
```

Comment:
I like this explanation. I missed this is earlier review (please also add it in the LESSONPLAN.md if you can)